### PR TITLE
Return appropriate error when 'notes' are not provided.

### LINKED ISSUE #2214: The CLI gives unhelpful error messages if --notes are not given
I've not identified the cause of the issue yet, but I noticed that the CLI cannot create updates:

```
$ bodhi updates new s2i-core-0-13.f27container --url http://localhost:6543
Required
You may not specify an empty list of builds.
ACL validation mechanism was unable to determine ACLs.
```

### PR DESCRIPTION
**Pull request by ghost**
fixes #2214

Signed-off-by: Vismay Golwala <vgolwala@redhat.com>

### CODE REVIEW

File: bodhi/client/__init__.py [hunk 1]
```diff
@@ -305,6 +305,10 @@ def new(user, password, url, **kwargs):
 
     kwargs['notes'] = _get_notes(**kwargs)
 
+    if not kwargs['notes']:
```
Comment (cverna) [inline bodhi/client/__init__.py:308]: we will need to check that also in the `edit` command

File: bodhi/tests/client/test___init__.py [hunk 2]
```diff
@@ -299,7 +299,7 @@ def test_severity_flag(self, send_request):
         result = runner.invoke(
             client.new,
             ['--user', 'bowlofeggs', '--password', 's3kr3t', '--autokarma', 'bodhi-2.2.4-1.el7',
-             '--severity', 'urgent'])
+             '--severity', 'urgent', '--notes', 'No description.'])
```
Comment (cverna) [inline bodhi/tests/client/test___init__.py:302]: I think we should test the command without --notes or --notes-file to make sure that we are getting the error message

**Review by cverna — CHANGES_REQUESTED** [review]

File: bodhi/server/schemas.py [hunk 3]
```diff
@@ -213,6 +213,7 @@ class SaveUpdateSchema(CSRFProtectedSchema, colander.MappingSchema):
     notes = colander.SchemaNode(
         colander.String(),
         validator=colander.Length(min=2),
+        missing_msg='Required description for the update.'
```
Comment (cverna) [inline bodhi/server/schemas.py]: Maybe we can change this to `A description is required for the update`. I find it easier to understand as an error message

File: bodhi/tests/client/test___init__.py (hunk 2 above)
Comment (ghost) [reply bodhi/tests/client/test___init__.py:302]: Thanks for pointing it out. Added a separate test for that!

**Review by cverna — APPROVED** [review]
LGTM

**Pull request merged**